playbooks/common/openshift-glusterfs/config.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Open firewall ports for GlusterFS
+  hosts: oo_glusterfs_to_config
+  vars:
+    os_firewall_allow:
+    - service: glusterfs_sshd
+      port: "2222/tcp"
+    - service: glusterfs_daemon
+      port: "24007/tcp"
+    - service: glusterfs_management
+      port: "24008/tcp"
+    - service: glusterfs_bricks
+      port: "49152-49251/tcp"
+  roles:
+  - os_firewall
+
+- name: Configure GlusterFS
+  hosts: oo_first_master
+  roles:
+  - role: openshift_storage_glusterfs
```

Comment:
We should add a `when` condition here so that `openshift_storage_glusterfs` is not applied to masters when glusterfs isn't being configured. Maybe:

```
when: groups.oo_glusterfs_to_config | default([]) | count > 0
```